pom.xml

```diff
@@ -520,6 +520,17 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>
```

Comment:
This plugin auto-promotes the staging repository in sonatype so a human doesn't have to login and manually promote the staging repository.